### DISCUSSION TITLE
Popover.Content not translating correctly

### DISCUSSION THREAD

**Discussion by raviteja83** [radix-ui/primitives#1665] [Help] [answered]
Any suggestions on how to narrow down on this?

"@radix-ui/react-popover": "1.0.0" - version

Broken popover. Expectation is popover end matches with the emoji
<img width="163" alt="image" src="https://user-images.githubusercontent.com/6763261/190322899-1d7d972e-c6ba-49a0-a2b9-84915ecd82c8.png">

Expectation:

<img width="457" alt="image" src="https://user-images.githubusercontent.com/6763261/190582145-d2334866-7e35-4348-9776-cfd65e067b55.png">

**Comment by benoitgrelard** [comment, marked as answer]
Hey @raviteja83,

I'm not sure what I'm supposed to be seeing here from the screenshot.
Would you have a sandbox to share so we can take a look at your issue?

**Reply by raviteja83** [reply]
@benoitgrelard Please refer to the sandbox below. The expectation is the end of popover content should align with end of smiley.
https://k9yzx7.csb.app/

**Reply by benoitgrelard** [reply]
You have `align="start"`, don't you want `align="end"` then?

**Reply by raviteja83** [reply]
It was behaving the same for both in this case. 

Note:  The emoji UI is attached to a div internally in the Picker component in the mounting `useEffect` if that helps.

**Reply by benoitgrelard** [reply]
I don't really follow the issue sorry.
Also I noticed you are using your own library (I assume built with Radix) so that makes it hard to debug anyway.
Are you able to reproduce an issue with Radix directly?

**Reply by raviteja83** [reply]
Really appreciate you taking the time to help out.
In the library, we are simply wrapping Popover with styled from stitches. 
I have tried with both `align="start"` and `align="end"` in the sandbox, but in both cases, the popover content starts at the end of the trigger.
If you see the expecation image in the question, that is what I am trying to achieve, instead what I am getting is the popover shifted to the right.

**Reply by benoitgrelard** [reply]
I understand.
The reason I asked for a barebones example is because there could be things happening within your lib/styles that throw it off, so it's very hard to take all that into considerations if we can't see it. Hence why I'm asking if you're able to reproduce your positioning issue in a simpler way.

As a start, here's the demo from the docs with `align="end"` which works fine: https://codesandbox.io/s/popover-align-end-2z9ysn?file=/App.js

**Reply by raviteja83** [reply]
I have updated the sandbox with the radix example. Seems like the issue might be with the Picker component. Can you point me in a direction, so I can figure out the issue.
https://k9yzx7.csb.app/

**Reply by raviteja83** [reply]
Seems to be working fine in the radix example after some css changes. Let me take a look again. Thanks for the assist. Marking as resolved.
